src/cryptography/x509/base.py

```diff
@@ -924,6 +924,8 @@ def sign(
         private_key: CertificateIssuerPrivateKeyTypes,
         algorithm: typing.Optional[_AllowedHashTypes],
         backend: typing.Any = None,
+        *,
+        rsa_padding: typing.Optional[padding.AsymmetricPadding] = None,
```

Comment:
Is there a reason not to use `Union[PSS, PKCS1v15]`? `OAEP` == guaranted type error.

(That interface seems like a mistake tbh)